### PR TITLE
BIB-441: Configuration setup

### PR DESCRIPTION
**Pull request by turegjorup**
Add configuration entities and editor

Note: 

- Actual endpoint is left for a later PR with auth etc.
- "Login metode(r)" mangler, har ikke oplysninger om hvilke/hvordan de skal defineres

### CODE REVIEW

File: src/Entity/Sip2User.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $username;
+
```
Comment (lilosti) [inline src/Entity/Sip2User.php:26]: A sip2 user consists of a username, a password and a location (=school). I think the school/location part is covered by the school enitity, but we are missing the password.

**Review by lilosti — CHANGES_REQUESTED** [review]
We need to add a sip2 password, as this is needed to connect to FBS.

File: src/Entity/Sip2User.php (hunk 1 above)
Comment (lilosti) [reply src/Entity/Sip2User.php:26]: I just disagreed with myself, we need the location on the sip2 user as well, as the "school" can have a different name (or want to show a different name on the screen) than what is used by FBS.

Comment (turegjorup) [reply src/Entity/Sip2User.php:26]: But the school entity only has the "common name", if it is part of the SIP2 login i would expect a formalized `school code` or similar?

Comment (turegjorup) [reply src/Entity/Sip2User.php:26]: Have added `$sip2InstitutionId` to `School` and `$password` to `Sip2User`

**Review by lilosti — APPROVED** [review]

**Pull request merged**